src/DataStructures/Tensor/Expressions/TensorIndex.hpp

```diff
@@ -161,6 +162,12 @@ static constexpr TensorIndex<7> ti_h{};
 static constexpr TensorIndex<
     TensorExpressions::TensorIndex_detail::upper_sentinel + 7>
     ti_H{};
+
+static constexpr TensorIndex<8> ti_t{};
```

Comment:
would it be better to give this the value right before the sentinel so that others can easily add more indices?